Basic-Audio-Transformer/test/test-chrome-firefox-safari-ie.js

```diff
@@ -0,0 +1,40 @@
+/* global browser describe it beforeAll $ */
+
+const assert = require('assert');
+
+describe('Basic Video Chat Test', () => {
+  beforeAll(() => {
+    browser.url('Basic Video Chat');
+  });
+
+  it('should have the right title', () => {
+    const title = browser.getTitle();
+    assert.equal(title, 'OpenTok Getting Started');
+  });
+
+  it('The publisher should load', () => {
+    const publisher = $('div.OT_publisher:not(.OT_loading) .OT_video-element');
+    publisher.waitForExist(15000);
+    browser.pause(2000);
+  });
+
+  describe('2 windows', () => {
```

Comment:
Nit: since this is an inner `describe` block, "2 windows" -> "when there are 2 windows"?